src/antelope/wallets/AntelopeWallets.ts

```diff
@@ -42,28 +42,19 @@ export class AntelopeWallets {
             // we try first the best solution which is taking the provider from the current authenticator
             const authenticator = account.authenticator as EVMAuthenticator;
             const provider = authenticator.web3Provider();
+            this.trace('getWeb3Provider authenticator.web3Provider() Success! (account.authenticator)', provider);
             return provider;
         } catch(e1) {
             this.trace('getWeb3Provider authenticator.web3Provider() Failed!', e1);
         }
 
```

Comment:
The second option was `window.ethereum` but now is removed. The problem with that option is that the `window.ethereum` may be present but the wallet may be not even opened, so the user is not logged and the chain is not set. In that case, the provider does not work so the option is discarded.